test/test.js

```diff
@@ -76,6 +76,14 @@ describe('hotkey', function () {
       assert.deepEqual(elementsActivated, [])
     })
 
+    it('triggers when user is focused on a input or textfield and hotkey includes modifier', function () {
+      setHTML(`
+      <button id="button1" data-hotkey="Meta+7">Button 1</button>
+      <input id="textfield" />`)
+      document.getElementById('textfield').dispatchEvent(new KeyboardEvent('keydown', {metaKey: true, key: '7'}))
+      assert.include(elementsActivated, 'button1')
+    })
```

Comment:
Copied the test above.

I confirmed this change in the browser, but I'm not sure this setup is acting as expected: 

```
document.getElementById('textfield').dispatchEvent
```